frontend/src/pages/landing/Landing.tsx

```diff
@@ -1,10 +1,70 @@
-import { redirect } from "react-router-dom";
+import { KanbanTemplate } from "@maykin-ui/admin-ui";
+import { useLoaderData } from "react-router-dom";
 
+import { KanbanCard } from "../../components/KanbanCard/KanbanCard";
+import {
+  DestructionList,
+  listDestructionLists,
+} from "../../lib/api/destructionLists";
+import { deslugify, timeAgo } from "../../lib/string";
 import "./Landing.css";
 
-/**
- * React Router loader.
- * @param request
- * TOOD: Requires destruction list lists endpoint.
- */
-export const landingLoader = () => redirect("/destruction-lists/create");
+interface LandingLoaderReturn {
+  [key: string]: DestructionList[];
+}
+
+export const landingLoader = async (): Promise<LandingLoaderReturn> => {
+  const lists = await listDestructionLists();
+  const statusMap: LandingLoaderReturn = {};
+
+  lists.forEach((list) => {
+    if (!statusMap[list.status]) {
+      statusMap[list.status] = [];
+    }
+    statusMap[list.status].push(list);
+  });
+
+  return statusMap;
+};
+
+// export const landingAction = async () => {
+//   await createDestructionList();
+//   return null;
+// };
+
+export const Landing = () => {
+  const lists = useLoaderData() as LandingLoaderReturn;
+
+  const constructComponentList = (lists: DestructionList[]) => {
+    const constructAssigneeNames = (
+      assignees: DestructionList["assignees"],
+    ) => {
+      const testAssignees = assignees.concat(assignees);
+      const sortedAssignees = testAssignees.sort((a, b) => a.order - b.order);
+      return sortedAssignees.map(
+        (assignee) =>
+          `${assignee.user.firstName} ${assignee.user.lastName} (${assignee.user.role.name})`,
+      );
+    };
+    return lists.map((list) => (
+      <KanbanCard
+        title={list.name}
+        days={timeAgo(list.created)}
+        assigneeNames={constructAssigneeNames(list.assignees)}
+        key={list.name}
+      />
+    ));
+  };
+
+  return (
+    <KanbanTemplate
+      kanbanProps={{
+        title: "Landing Page",
+        componentList: Object.keys(lists).map((status) => ({
+          title: deslugify(status),
```

Comment:
Maybe instead of this complex deslugify process, we could just have a hardcoded mapping for now. I am also thinking that the statuses that we currently have don't really make much sense :thinking: 
And maybe there could be an endpoint in the backend that returns the possible statuses and their label.
For now I would just go for the hardcoded mapping.

Reply:
Talked in call about it. I will hard-code them for now with a mapping and later on we will get an API endpoint for this.